forwards_paper.tex

```diff
@@ -165,21 +175,24 @@ \section*{Introduction}
 these tools may prove more widely useful.
 
 
-%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%
 \section*{Results}
 
 
 Below, we first describe the conceptual and algorithmic foundations for the method:
-(a) the new \emph{tables} format for storing tree sequences;
-(b) an algorithm to output tree sequence information on the fly
+(a) a format, implemented in the \msprime{} Python API,
+for storing tree sequences in several \emph{tables};
+(b) an algorithm, \algref{W}, to record tree sequence information into these tables on the fly
     from a forwards-time simulation;
 and (c) a method to \emph{simplify} a tree sequence, i.e., remove redundant information.
-This is followed by benchmarking comparisons
-with real, forwards-time simulations.
+We then describe a method that combines (b) and (d) to allows efficiently record simplified a tree sequence,
+and present analyses of its run time complexity.
+This is followed by benchmarking comparisons between two implementations of our method,
+each using previously published forwards-time simulation libraries.
 
 
 %%%%%%
-\subsection*{Recording a tree sequence}
+\subsection*{Storing a tree sequence}
```

Comment:
I like "recording", because that's the first what we do with this - record the output from the fwds-time sim.  We do also use the format to store tree sequences for the future (more or less?  not sure, actually how it differs from the binary storage).  But the main point for me is that this is the data format that gets acted on, processed, passed around.

Reply:
What about 'Describing?' As it stands the first and second sections are both 'Recording' one of them is action --- recording on the fly. The other is a description of the static data that describes a tree seq

Reply:
Maybe this first one could be "Working with tree sequences through tables" or some such?